Makefile

```diff
@@ -23,7 +23,7 @@ target/bundle/index.html:
 
 target/bundle/main.js: target/public/cljs-out/prod-main.js
 	mkdir -p target/bundle
-	cp target/public/cljs-out/prod-main.js target/bundle/main.js
+	cp target/public/cljs-out/prod/main_bundle.js target/bundle/main.js
```

Comment:
Interesting, so you do the explicit webpack thing I did on that branch? Was I wrong and it was actually _not_ using webpack until you did this and added `^{:auto-bundle :webpack}`?

Reply:
it WAS running a webpack bundle, its just the requirements for looking for those resources changed in the upgrade

Reply:
Right.. I think something confusing me was that in the (default) webpack setup it doesn't use the extra dir and `_bundle` postfix. This seems to be true even on the latest fighweel.